src/org/opensolaris/opengrok/history/MercurialRepository.java

```diff
@@ -613,70 +561,23 @@ boolean hasFileBasedTags() {
     }
 
     @Override
-    protected void buildTagList(File directory) {
+    protected void buildTagList(File directory, boolean interactive) {
```

Comment:
Should new `interactive` be used in the method?

Reply:
ditto